core/cloudflow-operator/src/main/scala/cloudflow/operator/DeploymentContext.scala

```diff
@@ -57,6 +57,10 @@ sealed trait ClusterType {
   override def toString = identifier
 }
 
+case object EksClusterType extends ClusterType {
+  override val identifier = "eks"
+}
+
```

Comment:
@lightbend/cloudflow-team  let's look at this because we were trying rally hard to remove this from the operator. The operator should not have to care which cluster type it is installed on so let's see if this is used anywhere and try to remove it ASAP.